### PR TITLE
Typo Error

### PR DESCRIPTION
**Pull request by aryamanpuri**

### CODE REVIEW

File: dev-docs/development-process.md [hunk 1]
```diff
@@ -34,11 +34,11 @@ export PATH=$PWD/node_modules/.bin:$PATH
 
 ## Hacking on the UI
 
-Taskcluster requires a Linux-like enviroment for development.
+Taskcluster requires a Linux-like environment for development.
 If you are developing on a Windows system, you will need to either
-* install [WSL](https://docs.microsoft.com/en-us/windows/wsl/install-win10);
-* use a virtual machine for development; or
-* install Linux as a second OS and boot into that.
+* Install [WSL](https://docs.microsoft.com/en-us/windows/wsl/install-win10)
```
Comment (helfi92) [inline dev-docs/development-process.md]:
```suggestion
* install [WSL](https://docs.microsoft.com/en-us/windows/wsl/install-win10)
```

File: dev-docs/development-process.md [hunk 2]
```diff
@@ -34,11 +34,11 @@ export PATH=$PWD/node_modules/.bin:$PATH
...
+* Use a virtual machine for development or
```
Comment (helfi92) [inline dev-docs/development-process.md]:
```suggestion
* use a virtual machine for development or
```

File: dev-docs/development-process.md [hunk 3]
```diff
@@ -34,11 +34,11 @@ export PATH=$PWD/node_modules/.bin:$PATH
...
+* Install Linux as a second OS and boot into that.
```
Comment (helfi92) [inline dev-docs/development-process.md]:
```suggestion
* install Linux as a second OS and boot into that.
```

**Comment by aryamanpuri** [comment]
@helfi92 I thought it was not intentional but by mistake.

**Pull request merged**